### PR TITLE
Vistatec/localisation checkin sleha15sp4

### PR DESCRIPTION
**Pull request by ptinley-vt**
### PR creator: Description
This pull request contains the localised XML and images for the SLE HA 15 SP4 components in locdrop2.

**SLEHA** 

**art.ha.install.quick_color**
de-de, es-es, fr-fr, it-it, ja-jp, pt_br, zh-cn, zh-tw (8 books)

**art_sle_ha_geo_quick**
de-de, es-es, fr-fr, it-it, ja-jp, pt_br, zh-cn, zh-tw (8 books)

**book.sleha_color**
ja-jp, zh-cn, zh-tw (3 books)

Regards,
Patrick

### CODE REVIEW

**Comment by jfaltenbacher** [comment]
optipng done for es-es, it-it, pt-br, zh-tw

**Comment by taroth21** [comment]
daps optipng done for remaining languages: de-de, fr-fr, ja-jp, zh-cn

**Comment by taroth21** [comment]
@ptinley-vt : Thank you very much! I have sent a mail last night where we reported some minor issues (mostly missing spaces within references before or after a booktitle (tagged with `<citetitle/>`), occurred in some languages only). Because those probably need to be fixed in the trans mem as well, we were not sure if to fix those in this PR already (or if you want to run an update from the trans mem).

**Review by taroth21 — CHANGES_REQUESTED** [review]
Thanks, looks good but for some minor issues - please see https://github.com/SUSE/doc-sleha/pull/235#issuecomment-1203553616

**Comment by ptinley-vt** [comment]
Hi Tanja,

Is it possible to provide me with a list of the files affected with this issue of missing spaces around <citetitle/>?
My first though is that it could be limited to CJK files, where space characters are not always used around cross-references.
I would like to see a list of the files effected of course.

Regards,
Patrick


Patrick Tinley
Senior Publishing Engineer | Vistatec
Vistatec House, 700 South Circular Road,
Kilmainham, Dublin 8, Ireland.
Tel: +353 1 416 8000
Email: ***@***.***
www.vistatec.com | ISO 9001 | ISO 13485 | ISO 17100 | ISO 27001 | ISO 18587
Think Global
 Vistatec Ltd.Registered in Ireland 268483. Registered Office,Vistatec House, 700, South Circular Road, Kilmainham. Dublin 8. Ireland.
The information contained in this message, including any accompanying documents, is confidential and is intended only for the addressee(s). 
​The unauthorized use, disclosure, copying, or alteration of this message is strictly forbidden. 
If you have received this message in error please notify the sender immediately.

File: l10n/sleha/de-de/xml/article_geo_clustering.xml [hunk 1]
```diff
@@ -0,0 +1,668 @@
...
+     Weitere Entscheidungen bezüglich der Einrichtung machen es möglicherweise erforderlich, weitere Ports zu öffnen (z. B. für DRBD oder Datenbankreplikation).
+    </para>
+   </listitem>
+  </itemizedlist>
+
+  <itemizedlist>
+  <title>Sonstige Anforderungen und Empfehlungen</title>
+  <listitem>
+   <para>
+    Alle Cluster-Knoten auf allen Sites sollten mit einem NTP-Server außerhalb des Clusters synchronisiert werden. Sie finden weitere Informationen hierzu im<link xlink:href="https://documentation.suse.com/sles-15/html/SLES-all/cha-ntp.html"><citetitle>Administrationshandbuch</citetitle> für SUSE Linux Enterprise Server <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></link>.
```
Comment (jfaltenbacher) [inline l10n/sleha/de-de/xml/article_geo_clustering.xml:177]: missing space here: 'im<link...'

File: l10n/sleha/de-de/xml/article_installation.xml [hunk 2]
```diff
@@ -0,0 +1,748 @@
...
+  </itemizedlist>
+
+  </sect2>
+  <sect2 xml:id="vl-ha-inst-quick-req-other">
+   <title>Sonstige Anforderungen und Empfehlungen</title>
+   <variablelist>
+    <varlistentry>
+     <term>Zeitsynchronisierung</term>  <listitem>
+   <para>
+     Cluster-Knoten müssen mit einem NTP-Server außerhalb des Clusters synchronisiert werden. Ab <phrase role="productname"><phrase os="sles">SUSE Linux Enterprise High Availability Extension</phrase></phrase> 15 ist chrony Standardimplementation von NTP. Sie finden weitere Informationen hierzu im<link xlink:href="https://documentation.suse.com/sles-15/html/SLES-all/cha-ntp.html"><citetitle>Administrationshandbuch</citetitle> für SUSE Linux Enterprise Server <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></link>.
```
Comment (jfaltenbacher) [inline l10n/sleha/de-de/xml/article_installation.xml:127]: missing space here: 'hierzu im<link...'

File: l10n/sleha/fr-fr/xml/article_installation.xml [hunk 3]
```diff
@@ -0,0 +1,748 @@
...
+  </variablelist>
+ </sect1>
+
+  <sect1 xml:id="sec-ha-inst-quick-installation">
+    <title>Installation de <phrase role="productname"><phrase os="sles">SUSE Linux Enterprise High Availability Extension</phrase></phrase></title>
+    <para>
+      Les paquetages de configuration et de gestion de grappe avec High Availability Extension sont inclus dans le modèle d&apos;installation <literal>haute disponibilité</literal> (appelé <literal>sles_ha</literal> dans la ligne de commande). Ce modèle est disponible uniquement si <phrase role="productname"><phrase os="sles">SUSE Linux Enterprise High Availability Extension</phrase></phrase> a été installé en tant qu&apos;extension de SUSE® Linux Enterprise Server.
+    </para>
+    <para>
+      Pour plus d&apos;informations sur l&apos;installation des extensions, reportez-vous au manuel<link xlink:href="https://documentation.suse.com/sles/html/SLES-all/cha-add-ons.html"><citetitle>Deployment Guide</citetitle> for SUSE Linux Enterprise Server <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></link> (Guide de déploiement de SUSE Linux Enterprise Server 15 SP4).
```
Comment (jfaltenbacher) [inline l10n/sleha/fr-fr/xml/article_installation.xml:257]: missing space here: 'au manuel<link...'

File: l10n/sleha/fr-fr/xml/article_installation.xml [hunk 4]
```diff
@@ -0,0 +1,748 @@
...
+    <varlistentry>
+     <term>Isolation de noeud/STONITH</term>
+     <listitem>  <para>
+      Pour éviter un scénario de <quote>vues de grappe divergentes</quote>, les grappes ont besoin d&apos;un mécanisme d&apos;isolation de noeud. Dans un scénario de vues de grappe divergentes, les noeuds de grappe sont divisés en deux groupes ou plus qui ne se connaissent pas (en raison d&apos;une défaillance matérielle ou logicielle, ou d&apos;une interruption de la connexion réseau). Un mécanisme d&apos;isolation isole le noeud en question (généralement en le réinitialisant ou en le mettant hors tension). Ce processus est également appelé STONITH (<quote>Shoot the other node in the head</quote>). Un mécanisme d&apos;isolation de noeud peut être un dispositif physique (un bouton marche/arrêt) ou un mécanisme de type SBD (STONITH par disque) associé à un système de surveillance (watchdog). L&apos;utilisation d&apos;un SBD nécessite un stockage partagé. 
+     </para> </listitem>
+    </varlistentry>
+   </variablelist>
+ </sect2>
+  <sect2 xml:id="il-ha-inst-quick-req-sw">
+   <title>CONFIGURATION LOGICIELLE REQUISE</title>
```
Comment (jfaltenbacher) [inline l10n/sleha/fr-fr/xml/article_installation.xml:103]: Uppercase letters

**Review by jfaltenbacher — CHANGES_REQUESTED** [review]
please see my comments regarding missing spaces and uppercase letters

File: l10n/sleha/de-de/xml/article_geo_clustering.xml (hunk 1 above)
Comment (ptinley-vt) [reply l10n/sleha/de-de/xml/article_geo_clustering.xml]: Fix now in XML on my local cloned repo and pushed to 'Vistatec/localisation checkin sleha15sp4' branch in repo.

File: l10n/sleha/de-de/xml/article_installation.xml (hunk 2 above)
Comment (ptinley-vt) [reply l10n/sleha/de-de/xml/article_installation.xml]: Fix now in XML on my local cloned repo and pushed to 'Vistatec/localisation checkin sleha15sp4' branch in repo.

File: l10n/sleha/fr-fr/xml/article_installation.xml (hunk 3 above)
Comment (ptinley-vt) [reply l10n/sleha/fr-fr/xml/article_installation.xml]: Fix now in XML on my local cloned repo and pushed to 'Vistatec/localisation checkin sleha15sp4' branch in repo.

File: l10n/sleha/fr-fr/xml/article_installation.xml (hunk 4 above)
Comment (ptinley-vt) [reply l10n/sleha/fr-fr/xml/article_installation.xml]: Fix now in XML on my local cloned repo and pushed to 'Vistatec/localisation checkin sleha15sp4' branch in repo.

File: l10n/sleha/de-de/xml/article_geo_clustering.xml (hunk 1 above)
Comment (ptinley-vt) [reply l10n/sleha/de-de/xml/article_geo_clustering.xml]: Updated in TM.
image

File: l10n/sleha/de-de/xml/article_installation.xml (hunk 2 above)
Comment (ptinley-vt) [reply l10n/sleha/de-de/xml/article_installation.xml]: image

Fixed now in the TM

image

File: l10n/sleha/fr-fr/xml/article_installation.xml (hunk 3 above)
Comment (ptinley-vt) [reply l10n/sleha/fr-fr/xml/article_installation.xml]: image

Fixed now in the TM

image

**Review by taroth21 — APPROVED** [review]
@ptinley-vt @jfaltenbacher : Thanks for fixing the issues. Looks good to me now!

**Pull request merged**